code/area.dm

```diff
@@ -2855,6 +2856,7 @@ TYPEINFO(/area/station/engine/substation)
 	luminosity = 1
 	force_fullbright = 1
 	requires_power = 0
+	minimaps_to_render_on = 0
```

Comment:
does this affect other maps tho

Reply:
The area in question is for a station level space structure, no rotation maps currently have this area and it'd make sense for it not to be on there anyway. This would effect older maps like trunkmap but they don't compile anyway.